### ISSUE TITLE
500 error code from consul should result in UNKNOWN

### ISSUE THREAD

**Issue by madAndroid** [sensu-plugins/sensu-plugins-consul#8]
We've discovered that if a consul agent receives a 500 during a check, the following is shown:
```
myserver.example.com/check_consul_leaders: Check failed to run: 500 Internal Server Error, ["/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/abstract_response.rb:74:in `return!'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/request.rb:495:in `process_result'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/request.rb:421:in `block in transmit'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/2.0.0/net/http.rb:852:in `start'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/request.rb:413:in `transmit'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/request.rb:176:in `execute'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugins-consul-0.0.6/bin/check-consul-leader.rb:74:in `run'", "/opt/sensu-plugins-omnibus/embedded/lib/ruby/gems/2.0.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]
```
Is it necessary that this returns `critical`? - it's a transient issue, and usually happens when a consul server has been rebooted; we rely on checks to notify us via pagerduty on critical thresholds, and this has fired off during an innocuous autoscaling event - it would be preferable for it to return UNKNOWN if it receives a 500 from consul.

**Comment by analytically** [comment]
Fixed in 0.0.7. Thanks for the PR!

**Comment by madAndroid** [comment]
thanks! :)
